roles/matrix-dynamic-dns/tasks/install.yml

```diff
@@ -0,0 +1,23 @@
+---
+
+- name: Ensure ddclient domain config exists
+  template:
+    src: "{{ role_path }}/templates/ddclient.conf.j2"
+    dest: "/etc/ddclient.conf"
+    mode: 0600
+  register: matrix_dynamic_dns_ddclient_domain_config
+
+- name: Ensure ddclient client config directory exists
+  file:
+    path: "/etc/default"
+    state: directory
+    mode: 0700
+    owner: "{{ user_username }}"
```

Comment:
What is `user_username` and `user_groupname`?

I wonder if creating/changing `/etc/default`, which already exists on a lot of (all) distros is a good idea?

On CentOS 7, `/etc/default` is owned by `root:root` with `755` permissions.